udata/templates/organization/sidebar-producer.html

```diff
@@ -14,6 +14,11 @@
 {% else %}
 <br/>
 {% endif %}
+<button type="button" class="btn btn-sm btn-block btn-left btn-warning" @click="$refs.discussions.start('')"
+    v-tooltip="'{{ _('Contact the producer') }}'" tooltip-placement="left">
+    <span class="fa fa-envelope-o"></span>
+    {{ _('Contact') }}
```

Comment:
The translation is missing (this is why I put a note, because the screenshot are not translated).